composer/workflows/airflow_db_cleanup.py

```diff
@@ -332,7 +332,7 @@ def cleanup_function(**context):
             dags = session.query(airflow_db_model.dag_id).distinct()
             session.commit()
 
-            list_dags = [str(list(dag)[0]) for dag in dags]
+            list_dags = [str(list(dag)[0]) for dag in dags] + [None]
```

Comment:
Is filter below "query.filter(airflow_db_model.dag_id == dag)" going to work for all supported databases?

I do not know if yes/no, but I believe there are some caveats that in some databases you should use .isNone or something to make it work.

Reply:
Yes, I tested it on few versions logs looks like this:
```
DELETION QUERY: SELECT log.id AS log_id, log.dttm AS log_dttm 
FROM log 
WHERE log.dttm <= %(dttm_1)s AND log.dag_id IS NULL
```
for None value and like this for regular value:
```
DELETION QUERY: SELECT log.id AS log_id, log.dttm AS log_dttm 
FROM log 
WHERE log.dttm <= %(dttm_1)s AND log.dag_id = %(dag_id_1)s
```
It is more sqlalchemy matter than Airflow itself.